src/assets/stackblitz/main.ts

```diff
@@ -86,6 +87,7 @@ export class DemoMaterialModule {}
     BrowserAnimationsModule,
     FormsModule,
     HttpModule,
```

Comment:
Should we remove `HttpModule`?

Reply:
`HttpModule` will be removed everywhere in a different PR.

Reply:
Good call - let's cleanup in another PR. Would like to at least just make sure this fix goes in without the risk of breaking other examples if I miss a case of one using `HttpModule`